kqueen/auth/common.py

```diff
@@ -9,21 +9,66 @@
 import importlib
 import logging
 
+config = current_config()
 logger = logging.getLogger('kqueen_api')
 
+"""
+Authentication Modules
+
+To define new module, need to specify it as dictionary, where:
+
+"auth_option_lower_case": {"engine": "EqualsToAuthClassName",
+                           "param": {
+                               "key": "value"
+                           }
+                           }
+"""
+
+AUTH_MODULES = {
+    "ldap": {
+        "engine": "LDAPAuth",
+        "param": {
+            "uri": config.get('LDAP_URI')
+        }
+    },
+    "local": {
+        "engine": "LocalAuth",
+        "param": {
+
+        }
+    }
+}
+
+
+def generate_auth_options(auth_list):
+    auth_options = {}
+
+    methods = auth_list.strip().split(',')
+    for m in methods:
+        if AUTH_MODULES[m]:
```

Comment:
`methods` is list of strings so you should do `if m in AUTH_MODULES`, this condition would actually fail with `KeyError` if `m` is not inside `AUTH_MODULES`

Reply:
good point thanks!

Reply:
done